crates/matrix-sdk-crypto/src/olm/group_sessions/outbound.rs

```diff
@@ -118,7 +117,7 @@ pub struct OutboundGroupSession {
     account_identity_keys: Arc<IdentityKeys>,
     session_id: Arc<str>,
     room_id: Arc<RoomId>,
-    pub(crate) creation_time: Arc<Instant>,
+    pub(crate) creation_time: Arc<SecondsSinceUnixEpoch>,
```

Comment:
Why is this in an `Arc`? It's cheaper to copy the `SecondsSinceUnixEpoch` (and I'm sure the same was true of `Instant`).